### DISCUSSION TITLE
Allow users to delete their profile on the website.

### DISCUSSION THREAD

**Discussion by NotAnderYT** [NamelessMC/Nameless#2230] [Ideas]
I think that there should be a feature where users can delete their profile through the profile settings on the website. It would require them to re log, just in case, then warn the user that all of their information would be gone and would not be recovered, and if they press OK, it would log them out and automatically start deleting their account, and then send a email notice to them letting the user know that the account was being deleted.

**Comment by Derkades** [comment]
https://github.com/NamelessMC/Nameless/issues/2071#issuecomment-768932556
